test/snippet/core/type_list/list_traits_find.cpp

```diff
@@ -0,0 +1,10 @@
+#include <seqan3/core/type_list/traits.hpp>
+
+int main()
+{
+    using list_t = seqan3::type_list<int, float, bool>;
+
+    // Double is not in t so find returns -1. However, bool is in the type list so find will return 2.
+    static_assert(seqan3::list_traits::find<double, list_t>, -1);
```

Comment:
```suggestion
    static_assert(seqan3::list_traits::find<double, list_t> == -1);
```
and everywhere else. Otherwise I think -1 is the message :thinking: This should not compile...